src/games/strategy/engine/pbem/TripleAForumPoster.java

```diff
@@ -0,0 +1,193 @@
+package games.strategy.engine.pbem;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.http.NameValuePair;
+import org.apache.http.client.CookieStore;
+import org.apache.http.client.config.CookieSpecs;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.protocol.HttpClientContext;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.mime.MultipartEntityBuilder;
+import org.apache.http.impl.client.BasicCookieStore;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.protocol.BasicHttpContext;
+import org.apache.http.protocol.HttpContext;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import games.strategy.debug.ClientLogger;
+import games.strategy.engine.framework.system.HttpProxy;
+import games.strategy.net.OpenFileUtility;
+import games.strategy.triplea.help.HelpSupport;
+import games.strategy.util.Util;
+
+public class TripleAForumPoster extends AbstractForumPoster {
+
+  private static final long serialVersionUID = -3380344469767981030L;
+
+  public static final String tripleAForumURL = "https://forums.triplea-game.org";
+
+
+  @Override
+  public boolean postTurnSummary(String summary, String subject) {
```

Comment:
This code is new, entirely written from scratch. Of course the other forum posters were a good example